src/Workspaces/VisualBasic/Portable/CodeCleanup/Providers/ReduceTokensCodeCleanupProvider.vb

```diff
@@ -60,24 +66,20 @@ Namespace Microsoft.CodeAnalysis.CodeCleanup.Providers
 
                     Case SyntaxKind.DecimalLiteralToken
                         ' Get the literal identifier text which needs to be pretty listed.
-                        Dim idText = literal.GetIdentifierText()
                         Dim value = DirectCast(literal.Value, Decimal)
 
                         If value = 0 Then
                             ' Overflow/underflow case or zero literal, skip pretty listing.
                             Return newNode
                         End If
-
                         ' Compiler has parsed the literal text as a decimal value, fetch the string representation of this value.
```

Comment:
It is our custom, documented by analogy with the C# coding guidelines, to always leave a blank line after an End If. Please restore it.